packages/core-data/src/utils/user-permissions.js

```diff
@@ -0,0 +1,40 @@
+export const ALLOWED_RESOURCE_ACTIONS = [
+	'create',
+	'read',
+	'update',
+	'delete',
+];
+
+export function getUserPermissionsFromResponse( response ) {
+	const permissions = {};
+
+	// Optional chaining operator is used here because the API requests don't
+	// return the expected result in the native version. Instead, API requests
```

Comment:
Made me re-read it to understand we're talking about the React Native implementation. 

```suggestion
	// return the expected result in the React native version. Instead, API requests
```

Reply:
The React Native special case was discussed in detail here: https://github.com/WordPress/gutenberg/pull/39144#issuecomment-1056701720

In short, RN response always have a parsed body and never include headers. Except when the response is an error.

Another thing that's hard to understand is this line:
```js
const allowedMethods = allowHeader?.allow || allowHeader || ''
```
In what case could `allowHeader` be an object with an `allow` field? This was added by @tyxla in a Lodash removal PR: https://github.com/WordPress/gutenberg/pull/43117/files#diff-fec6390e24332dec4a0c7c74f9f7b9a015ce191e0e6348361ead793ceda3991dR319

The original code was `includes( allowHeader, method )`. I still don't understand accessing the `.allow` field.

Reply:
This appears to be a mistake 🤔 Which seems to be additionally covered by a pre-existing test fixture. Let me fix.

Reply:
Fix in #63615

Reply:
Landed. Feel free to rebase @Mamaduka

Reply:
Done ✅